### PR TITLE
Feature/72 city picker for new user

### PR DESCRIPTION
**Pull request by Rodrigo-Schar**
- added city picker
- added city picker for settings and launcher
- fixed get country model
- merge develop
- added change map based on city
- added cities list and fix design
- added UnitTest and corrections
- removed import
- fix images/icons name

### CODE REVIEW

File: RoutesApp_ios/modules/citypicker/controllers/CityPickerViewController.swift [hunk 1]
```diff
@@ -0,0 +1,125 @@
+//
+//  CityPickerViewController.swift
+//  RoutesApp_ios
+//
+//  Created by admin on 8/22/22.
+//
+
+import UIKit
+import SVProgressHUD
+import SwiftUI
```
Comment (jimibilibob) [inline RoutesApp_ios/modules/citypicker/controllers/CityPickerViewController.swift]: Why are you using SwiftUI?

Comment (Rodrigo-Schar) [reply RoutesApp_ios/modules/citypicker/controllers/CityPickerViewController.swift]: import removed!

File: RoutesApp_ios/Assets.xcassets/Icons/arrow-left-primary.imageset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Arrow 1.svg",
```
Comment (jmacboy) [inline RoutesApp_ios/Assets.xcassets/Icons/arrow-left-primary.imageset/Contents.json]: use more descriptive names for your files, Arrow 1.svg should be arrow-left-primary.svg

File: RoutesApp_ios/Assets.xcassets/Icons/arrow-right.imageset/Contents.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "Arrow 5.svg",
```
Comment (jmacboy) [inline RoutesApp_ios/Assets.xcassets/Icons/arrow-right.imageset/Contents.json]: same for filename here

File: RoutesApp_ios/Assets.xcassets/Images/city-picker-background.imageset/Contents.json [hunk 4]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "image 7.svg",
```
Comment (jmacboy) [inline RoutesApp_ios/Assets.xcassets/Images/city-picker-background.imageset/Contents.json]: also the name of this image

File: RoutesApp_ios/Assets.xcassets/Images/city-picker-emoji.imageset/Contents.json [hunk 5]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "exito (2) 1.png",
```
Comment (jmacboy) [inline RoutesApp_ios/Assets.xcassets/Images/city-picker-emoji.imageset/Contents.json]: and this one

File: RoutesApp_ios/Assets.xcassets/Images/city-picker-world.imageset/Contents.json [hunk 6]
```diff
@@ -0,0 +1,21 @@
+{
+  "images" : [
+    {
+      "filename" : "image 5.svg",
```
Comment (jmacboy) [inline RoutesApp_ios/Assets.xcassets/Images/city-picker-world.imageset/Contents.json]: also this one

**Review by jmacboy — CHANGES_REQUESTED** [review]

**Review by jmacboy — APPROVED** [review]

**Review by jimibilibob — APPROVED** [review]

**Pull request merged**